pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -196,8 +197,17 @@ func (c *Cluster) failMon(monCount, desiredMonCount int, name string) {
 		}
 	} else {
 		// bring up a new mon to replace the unhealthy mon
-		if err := c.failoverMon(name); err != nil {
-			logger.Errorf("failed to failover mon %q. %v", name, err)
+		// only possible if we have enough "ready nodes"
+		if readyNodes, err := k8sutil.GetKubernetesReadyNodes(c.context.Clientset); err != nil {
```

Comment:
If `allowMultiplePerNode: true` in the cluster CR, we should skip this check. We need this for testing.

Reply:
This check for nodes is just a nice-to-have. Let's proceed with the failover even if `GetKubernetesReadyNodes()` fails.